doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -74,11 +74,14 @@ The picture below highlights the PMON vertical and its association with other lo
 #### Use case
 * Switch: Day0 setup, Maintenance, RMA
 * DPU: Day0, Day1-N Provisioning, Maintenance, RMA
+#### DPU dark mode
+* When a DPU is in dark mode, it remains powered down and does not consume any power.
+* Dark mode is enabled by default, but can be configured through the platform's config_db if needed.
```

Comment:
It will be appropriate to say "When a DPU is in dark mode, it remains powered down and does not consume power"
"Dark mode is enabled by default, but the platform can be configure to boot in light up mode with one or more DPUs"